client/my-sites/plans-features-main/index.tsx

```diff
@@ -240,6 +240,7 @@ const PlansFeaturesMain = ( {
 	const { setShowDomainUpsellDialog } = useDispatch( WpcomPlansUI.store );
 	const domainFromHomeUpsellFlow = useSelector( getDomainFromHomeUpsellInQuery );
 	const showUpgradeableStorage = config.isEnabled( 'plans/upgradeable-storage' );
+	const enableTermSavingsPriceDisplay = isEnabled( 'plans/term-savings-price-display' );
```

Comment:
I'm looking at the changes again, and I think actually renaming everywhere to something like `reflectStorageSelectionInPlanPrices` (or something else) makes sense. We chatted about doing this change only to the data-store hook in our [other discussion](https://github.com/Automattic/wp-calypso/pull/96520/files#r1848663632), but looking again, there's really nothing about these changes that directly reflect the "term savings price display". It really is about whether to reflect or not reflect the storage selection in the price. So I think I'd keep it isolated to that, and also rename the feature flag to match better. The "term savings price display" is a different thing to me.

Reply:
[32de474](https://github.com/Automattic/wp-calypso/pull/96520/commits/32de474bc4f1d0091a0aae0be3c5b509d65d05d0)